src/main/resources/assets/api-docs.yaml

```diff
@@ -1863,6 +1863,33 @@ paths:
             type: array
             items:
               type: string
+  /api/dataset/test:
+    post:
+      summary: Creates the Dataset from JSON
+      description: Creates the Dataset from JSON (WIP)
+      consumes:
+        - application/json
+      parameters:
```

Comment:
Added params so that you can test it when the server is running and see the mostly empty Dataset returned in a 201 response